lib/Server.js

```diff
@@ -120,6 +119,7 @@ function Server(compiler, options) {
 
 		proxy: function() {
 			if (options.proxy) {
+				var proxy = new httpProxy.createProxyServer({secure: options.secureProxy});
```

Comment:
Good point, @sokra. What do you think of just always passing `secure: false`, to support valid certificates that don't link to the Node.js certificate chain?